app/controllers/stories_controller.rb

```diff
@@ -294,7 +294,6 @@ def assign_article_show_variables
     end
 
     @comments_to_show_count = @article.cached_tag_list_array.include?("discuss") ? 50 : 30
-    assign_second_and_third_user
```

Comment:
Just looking at this again, and I'm curious: why does this PR remove this step of assigning these users? I would think that refactor would still keep around this step of assigning coauthors (but move from `second_user_id` and `third_user_id` to instead be `co_author_ids`) here too, right? Unless there is something I am not seeing in which case please do point it out to me! :)

Reply:
Great question, @vaidehijoshi! At the time that I had removed this, it looked like `co_author_ids` was assigned elsewhere, but over the past two days as I've been addressing outstanding PR comments, I'm unsure of whether or not what I had initially thought is true -- I'm unable to find where they would be assigned. 🙈 All of that being said, I'm reverting this change and adding back the assignment of `co_author_ids` to the `Stories::Controller`. 😊